docs/deploy/prometheus.rst

```diff
@@ -29,3 +29,29 @@ If the ``prometheus-client-nginx`` state file applies to the target, `Node Expor
 .. note::
 
    Bytemark assigns hostnames like ``<server>.<group>.opencontracting.uk0.bigv.io`` to its servers, and implements wildcard DNS for any subdomains. By default, Node Exporter is served from ``prom-client.<hostname>`` on port 80, which works for Bytemark servers without additional configuration. For Hetzner servers, additional configuration is needed. Instead of adding a DNS entry and setting the ``prometheus.client_fqdn`` variable, we simply set the ``prometheus.client_port`` variable and access Node Exporter by the server's IP address.
+
+
+Upgrading Prometheus Software
+-----------------------------
+
+We lock to set versions of the Prometheus software for consistent servers.
+
+We set the versions in variables in the ``pillar/private/prometheus_pillar.sls`` file:
+
+* ``server_prometheus_version``
+* ``server_alertmanager_version``
+* ``node_exporter_version``
+
+Upgrading to the latest versions should be done periodically.
+
+Before upgrading:
+
+* Check the release logs for any breaking changes that would affect our setup.
+* Consider running the upgrade on a virtual machine to test locally.
```

Comment:
"Running the upgrade" makes me think there's some additional step – but there is none in this case. "Consider deploying to a virtual machine to test locally" is more clear (we can link to the heading about how to do that with `:ref:`).